README.md

```diff
@@ -1,6 +1,7 @@
-# W3C Web of Things - Thing Description Playground
+<img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width=300px>
 
-Try it online at [http://plugfest.thingweb.io/playground/](http://plugfest.thingweb.io/playground/)
+## Thing Description Playground
```

Comment:
Does it make sense to add on each root readme the "Thingweb" prefix like the following?

`## Thingweb - Thing Description Playground`

OR is the logo enough?